Create ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md (#2649)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md

---
date: ""

> title: [UI] Fail firmware update process if device reports unexpected UPDATE_RESULT [GRAFT] (#7500) [GRAFT][release/cd] (#7580)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Firmware update process fails if device reports unexpected UPDATE_RESULT
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4050
version: 1021.4.2
---

> [UI] Fail firmware update process if device reports unexpected UPDATE_RESULT [GRAFT] (#7500) [GRAFT][release/cd] (#7580)

Author: The Claude AI suggested to use the following body:
```suggestion
In some cases, devices reported an unexpected UPDATE_RESULT status during the firmware update process, but the update was still treated as successful. With this change, the firmware update process will now fail if a device reports an unexpected UPDATE_RESULT status. This prevents incorrectly treating a failed update as successful, ensuring the device firmware is in the expected state after an update.
```

---
date: ""
title: Terminate the firmware update process if an unexpected situation is detected
product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4050
version: 1021.4.2
---

> Before the change if a device reported an unexpected status or result during a firmware update,

Reviewer: ```suggestion
Previously, if a device reported an unexpected status or result during a firmware update,
the process simply waited for a valid state. Now, you can configure LWM2M devices to fail the firmware update
process when an unexpected situation is detected.
```

---
date: ""
title: Terminate the firmware update process if an unexpected situation is detected
product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4050
version: 1021.4.2
---
Previously, if a device reported an unexpected status or result during a firmware update,
the process simply waited for a valid state. Now, you can configure LWM2M devices to fail the firmware update
process when an unexpected situation is detected.

> process when an unexpected situation is detected.

Reviewer: ```suggestion
```

---
date: ""
title: Terminate the firmware update process if an unexpected situation is detected
product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4050
version: 1021.4.2
---
Previously, if a device reported an unexpected status or result during a firmware update,
the process simply waited for a valid state. Now, you can configure LWM2M devices to fail the firmware update
process when an unexpected situation is detected.

> the process would simply wait for a valid state. Now, you can configure LWM2M devices to fail the firmware update

Reviewer: ```suggestion
```
